src/System.Diagnostics.Process/src/System.Diagnostics.Process.builds

```diff
@@ -11,6 +11,9 @@
     <Project Include="System.Diagnostics.Process.csproj">
       <OSGroup>Windows_NT</OSGroup>
     </Project>
+    <Project Include="System.Diagnostics.Process.csproj">
+      <TargetGroup>net461</TargetGroup>
+    </Project>
     <Project Include="facade\System.Diagnostics.Process.csproj">
```

Comment:
Just curious, how would that be any different from this change? Although we are building the csproj as a partial facade, we are not including any source files for net461, so I would assume genfacade.exe should create the same facade which we would check-in. Isn't it?